### PR TITLE
Chocolate bonbons

### PR DESCRIPTION
**Pull request by aleene**
### What
There was a missing ecoscore here.

It fits however: https://www.agribalyse.fr/app/aliments/31091#Bonbon_/_bouchée_au_chocolat_fourrage_gaufrettes_/_biscuit

Which seems a pretty large category.

### Part of
- #5520

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -27286,29 +27286,11 @@ nl:Pralines, chocolades snoepjes
 pt:Bombons
 ru:Конфеты
 wikidata:en:Q3272827
+agribalyse_food_code:en:31091
+ciqual_food_code:en:31091
+ciqual_food_name:en:Chocolate confectionery, filled with wafer
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:27291]: this CIQUAL category is for chocolate confectionery with a wafer filling, it's not any kind of candy, it does not work here.

**Review by stephanegigandet — CHANGES_REQUESTED** [review]

File: taxonomies/categories.txt (hunk 1 above)
Comment (aleene) [reply taxonomies/categories.txt:27291]: Maybe more this one: https://www.agribalyse.fr/app/aliments/31063#Bouchée_chocolat_fourrage_fruits_à_coques_et/ou_praliné

Comment (stephanegigandet) [reply taxonomies/categories.txt:27291]: in fact my mistake, Github is not showing the lines above (only 3), I didn't realize it was for chocolate bonbons.

so it's a good category. Make it a proxy maybe.

```suggestion
ciqual_proxy_food_name:en:Chocolate confectionery, filled with wafer
```

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -27286,29 +27286,11 @@ nl:Pralines, chocolades snoepjes
 pt:Bombons
 ru:Конфеты
 wikidata:en:Q3272827
+agribalyse_food_code:en:31091
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:27289]:
```suggestion
agribalyse_proxy_food_code:en:31091
```

File: taxonomies/categories.txt [hunk 3]
```diff
@@ -27286,29 +27286,11 @@ nl:Pralines, chocolades snoepjes
...
+ciqual_food_code:en:31091
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:27290]:
```suggestion
ciqual_proxy_food_code:en:31091
```

**Pull request closed without merging**